eventsessions/views.py

```
@@ -0,0 +1,144 @@
+from eventsessions.models import Session
+from eventsessions.serializers import SessionSerializer
+
+
+@extend_schema(
+    responses=SessionSerializer, request=SessionSerializer, tags=["Event Sessions"]
+)
+class SessionListCreateView(APIView):
+    """View for listing all sessions of an event and creating new sessions."""
+
+    serializer_class = SessionSerializer
+    permission_classes = [IsOrganizationAdminOrOrganizer]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)